nodestream/pipeline/scope_config.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+
+from ..file_io import LoadsFromYamlFile
+
+
+class ScopeConfig(LoadsFromYamlFile):
```

Comment:
We can remove the implementations of `__init__` and and `__eq__` if we make this a [dataclass](https://docs.python.org/3/library/dataclasses.html).